FullyNoded/View Controllers/Home/Home Screen/MainMenuViewController.swift

```diff
@@ -118,7 +118,7 @@ class MainMenuViewController: UIViewController, UITableViewDelegate, UITableView
                     if hostname != nil {
                         hostname = hostname?.replacingOccurrences(of: "\n", with: "")
                         DispatchQueue.main.async { [weak self] in
-                            self?.host = "\(prefix)://\(rpcusername):\(rpcpassword)@\(hostname!):11221/?label=\(nodeStruct.label.replacingOccurrences(of: " ", with: "%20"))"
+                            self?.host = "\(prefix)://\(rpcusername):\(rpcpassword)@\(hostname!):11221/?label=\(nodeStruct.walletLabel.replacingOccurrences(of: " ", with: "%20"))"
```

Comment:
this was a bug, it should be `label`, causes a crash as is

Reply:
Not sure what happened here.